README.md

```diff
@@ -1,19 +1,67 @@
 # Pulumi CDK Adapter
 
-The Pulumi CDK Adapter is a library that enables
-[Pulumi](https://github.com/pulumi/pulumi) programs to use [AWS
-CDK](https://github.com/aws/aws-cdk) constructs.
+The pulumi-cdk library provides access to the many high-level libraries ('constructs') built by service
+teams at AWS and by [the AWS CDK community](https://constructs.dev/).
 
-The adapter allows writing AWS CDK code as part of an AWS CDK Stack inside a
-Pulumi program, and having the resulting AWS resources be deployed and managed
-via Pulumi.  Outputs of resources defined in a Pulumi program can be passed
-into AWS CDK Constructs, and outputs from AWS CDK stacks can be used as inputs
-to other Pulumi resources.
+The adapter allows writing [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/home.html) code inside a
+Pulumi program, and having the resulting AWS resources be deployed and managed via Pulumi. Pulumi and CDK resources can
+seamlessly interact with each other. Outputs of resources defined in a Pulumi program can be passed into AWS CDK Constructs,
+and outputs from AWS CDK stacks can be used as inputs to other Pulumi resources.
 
-For example, to construct an [AWS AppRunner `Service`
-resource](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-apprunner-alpha-readme.html)
-from within a Pulumi program, and export the resulting service's URL as as
-Pulumi Stack Output you write the following:
+## Getting Started
+
+To get started with CDK on Pulumi first [download and install Pulumi](https://www.pulumi.com/docs/install/), and [configure it to work with your AWS account](https://www.pulumi.com/registry/packages/aws/installation-configuration/).
+Next, create a Pulumi TypeScript project, install the required packages, and
+ensure you have configured the AWS providers.
+
+```bash
+$ pulumi new aws-typescript
+$ npm install @pulumi/aws @pulumi/aws-native @pulumi/cdk @pulumi/docker-build @pulumi/pulumi aws-cdk-lib
+$ pulumi config set aws-native:region us-east-2
+$ pulumi config set aws:region us-east-2
+```
+
+### cdk.json File
+
+Settings for CDK applications are typically stored in a `cdk.json` file at the
+root of the project. It is recommended that you create one and populate it with
+a couple of settings.
+
+- `app`: This setting controls what command to run in order to synthesize the
+    CDK application.
+- `output`: The directory location where the CDK output will be placed. If this
+    is not specified it will be placed in a system temporary directory and could
+    cause issues with Docker assets.
```

Comment:
I'd point out what issues exactly, IMO this makes it sound like something will go wrong or the docker images will get mangled.